### PR TITLE
feat(v4.1): handling configuration for websocket enabled by default (WPB-967)

### PR DESCRIPTION
**Pull request by yamilmedina**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-967" title="WPB-967" target="_blank">TaskWPB-967</a>  Websocket should be turned on by default on bund apps
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

We need to enable the websocket by default on devices that doesn't support google play services. ie. col1/3/fdroid builds

### Solutions

When storing the session of the user, persist this config accordingly by configuration given by `KaliumConfig`
So we use the checks to know if this environment has google playservices and persist the account with this flag.

### Dependencies (Optional)

- https://github.com/wireapp/kalium/pull/1852

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1899/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.6(6941244)(6941244)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.6(6941244)(6941244)-staging-compat.apk)

**Review by trOnk12 — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt [hunk 1]
```diff
@@ -56,7 +58,8 @@ class KaliumConfigsModule {
             guestRoomLink = BuildConfig.ENABLE_GUEST_ROOM_LINK,
             wipeOnCookieInvalid = BuildConfig.WIPE_ON_COOKIE_INVALID,
             wipeOnDeviceRemoval = BuildConfig.WIPE_ON_DEVICE_REMOVAL,
-            wipeOnRootedDevice = BuildConfig.WIPE_ON_ROOTED_DEVICE
+            wipeOnRootedDevice = BuildConfig.WIPE_ON_ROOTED_DEVICE,
+            isWebSocketEnabledByDefault = runBlocking { !context.isGoogleServicesAvailable() }
```
Comment (MohamadJaara) [inline app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt:62]: We can replace it with a build time flag, and set it to true for specific builds

**Review by Garzas — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1899/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.6(6942263)(6942263)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.6(6942263)(6942263)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt (hunk 1 above)
Comment (yamilmedina) [reply app/src/main/kotlin/com/wire/android/di/KaliumConfigsModule.kt:62]: I also thought that, but after discussing with Marco/Leen and pairing the old app behavior, this is the way they decided to go. funny fact the old app also relied in a build config, but was changed overtime.

**Pull request merged**